### DISCUSSION TITLE
How to handle page-size changing event on NDataTable?

### DISCUSSION THREAD

**Discussion by Asgen** [tusen-ai/naive-ui#1659] [Q&A] [answered]
There is a table
`<n-data-table
    class="customers-table"
    ref="table"
    remote
    :bordered="false"
    :columns="columns"
    :data="data"
    :loading="loading"
    :pagination="pagination"
    :row-key="rowKey"
    @update:page="handlePageChange"
  />`

Pagination objects looks like this
`{
      page: 1,
      pageCount: 1,
      pageSize: store.state.paginationParam.per,
      showSizePicker: true,
      itemCount: 0,
      pageSizes: pageSizes,
      prefix: (info: PaginationInfo) =>
        h("div", [
          h(
            "p",
            `${i18n.global.t("Customers.Table.AllClients")}: ${info.itemCount}`
          ),
          h(
            "p",
            `${i18n.global.t("Customers.Table.AllOrders")}: ${info.itemCount}`
          ),
        ]),
    }`
How can I handle page-size event in this case?

**Comment by Asgen** [comment, marked as answer]
Oh, it's just a listener on <data-table> @on-update:page-size

**Reply by 07akioni** [reply]
Yes
